### DISCUSSION TITLE
Different präsentation in result- and filterwebpart

### DISCUSSION THREAD

**Discussion by Waldmann30** [microsoft-search/pnp-modern-search#3464] [Q&A]
We have different präsentation in the result- and filterwebpart.
But not with all documents.
In the result web part, the metadata is not displayed for some documents. (Image1)
after filtering, all the information are there (Image2)
**Image1:**
![Image1_resultweppart](https://github.com/microsoft-search/pnp-modern-search/assets/157123123/6fb2338e-d798-4c3e-9c6b-f3d83787ca4e)
**Image2:**
![Image2_filtering](https://github.com/microsoft-search/pnp-modern-search/assets/157123123/fa1f48ef-d707-4b95-ab1e-e17f40b5fc9e)
![Image2 2_filtering](https://github.com/microsoft-search/pnp-modern-search/assets/157123123/bddff356-d40f-47c2-a665-db387379a808)
Is there someone for ideas :-)
Thank you in advance.

**Comment by kasperbolarsen** [comment]
I assume you are running version 4.10.1 of the PnP Modern Search web parts ?
You are using SharePoint as the Data Source?

Can you share the query being executed in the two cases, ( open Browser Tools, Network, filter for postquery and go to the payload tab, expand the request node)

**Reply by Waldmann30** [reply]
Hello Kasperbolarsen.

I use the Version: 4.9.3 and yes we use SharePoint as the Data Source.
Look like we found the solution. An end user create a Library, and copied the duplicates into this library without metadatas :-(
I didn't see it properly
